shared/src/common/enums/task-note/task-note-validation-rule.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum TaskNoteValidationRule {
```

Comment:
```suggestion
enum TaskNoteValidationMessage {
```
don't forget about file name